lib/resources/command.rb

```diff
@@ -28,6 +28,9 @@ def initialize(cmd)
     end
 
     def result
+      if @command.nil?
+        raise 'InSpec `command` was called with `nil` as the argument. This is not supported. Please provide a valid command instead.'
```

Comment:
This feels like it belongs in `#initialize` since no method in this resource should work / will work with a `nil` command. Is there a technical reason you put it here instead of in the constructor?